src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -157,12 +179,52 @@ private static ReportType GetReportType(bool isDeprecated, bool isOutdated, bool
             throw new ArgumentException(string.Format(Strings.ListPkg_InvalidOptions));
         }
 
+        private static (IReportRenderer, ReportOutputFormat) GetOutputType(string outputFormatOption, string outputVersionOption)
+        {
+            ReportOutputFormat outputFormat = ReportOutputFormat.Console;
+            if (!string.IsNullOrEmpty(outputFormatOption))
+            {
+                try
+                {
+                    outputFormat = EnumExtensions.GetValueFromName<ReportOutputFormat>(outputFormatOption);
+                }
+                catch (ArgumentException)
+                {
+                    string currentlySupportedFormat = "console, json";
```

Comment:
This should be generated by looking at all the values of the enum, not by hardcoding it.

Reply:
This one is temporary; I want feedback on main design.